docs/src/content/docs/basic-concepts.mdx

```diff
@@ -203,3 +203,5 @@ Providers offer several benefits:
 - Provide a simpler, more focused API than raw contexts
 - Keep configuration separate from business logic
 - Enable dependency injection for testing and development
+
+For more details on providers, see the [Providers](../concepts/providers) page.
```

Comment:
Without a preview I'm not 100% certain but I think this should be `../../concepts/providers`

Reply:
I tested this link in dev and it worked so we should be all good.